backend/handler.go

```diff
@@ -64,6 +65,7 @@ func NewHandler(repo *sqlRepo, config *Config) (http.Handler, error) {
 	r.POST("/grant-access", h.grantAccess)
 	r.GET("/flag", h.getFlag)
 	r.POST("/flag", h.submitFlag)
+	r.GET("/static/*filepath", h.static)
```

Comment:
Here I register the files using an embed filesystem.
So all the files we serve to the end-user will be served from within the compiled binary. This should help mitigate deployment problems